src/externalized/DefaultContentManager.cc

```diff
@@ -1185,18 +1193,33 @@ bool DefaultContentManager::loadMercsData()
 	}
 	MercProfileInfo::validateData(m_mercProfileInfo);
 
-	json = readJsonDataFileWithSchema("mercs-relations.json");
-	for (auto& element : json.toVec()) {
-		JsonObject reader = element.toObject();
-		const MercProfileInfo* inf = this->getMercProfileInfoByName(reader.GetString("profile"));
-		MercProfile::deserializeStructRelations(temp_mercStructs[inf->profileID].get(), reader, this);
-	}
-	for (auto& element : temp_mercStructs) {
-		if (element != nullptr) {
-			m_mercStructs.push_back( std::make_unique<const MERCPROFILESTRUCT>( *element ) );
+	if (!doesGameResBelongToMod(BINARYDATADIR "/prof.dat")) {
```

Comment:
Not sure if this is the best approach to do this. We could also always load `prof.dat` and only use it if no JSON data is provided. Similar to what is done with `VanillaItemStrings`, where the data is passed to the individual JSON deserialization methods of the items.